cmd/traffic_manager/traffic_manager.cc

```diff
@@ -418,7 +418,7 @@ main(int argc, const char **argv)
 
   // Before accessing file system initialize Layout engine
   Layout::create();
-  mgmt_path = Layout::get()->sysconfdir;
+  mgmt_path = Layout::get()->sysconfdir.c_str();
```

Comment:
This case is not temporary lifetime issue. It depends on the layout class directories as before.